build/ci/evergreen.yml

```diff
@@ -857,6 +857,27 @@ tasks:
           E2E_TAGS: atlas,datafederation,privatenetwork
           E2E_TEST_BUCKET: ${e2e_test_bucket}
           E2E_CLOUD_ROLE_ID: ${e2e_cloud_role_id}
+  # datafederation requires cloud provider role authentication and an s3 bucket created
+  - name: atlas_datafederation_querylimits_e2e
+    tags: ["e2e","datafederation","atlas"]
```

Comment:
question: should "querylimits" tag be in this list?
in the e2e we have `//go:build e2e || (atlas && datafederation && querylimits)`